packages/components/upload/src/use-handlers.ts

```diff
@@ -53,7 +53,7 @@ export const useHandlers = (
 
   function removeFile(file: UploadFile) {
     uploadFiles.value = uploadFiles.value.filter(
-      (uploadFile) => uploadFile !== file
+      (f) => JSON.stringify(f) !== JSON.stringify(file)
```

Comment:
```suggestion
      (uploadFile) => uploadFile.uid !== file.uid
```
It is more reasonable to use uid to judge.

Reply:
More reasonable and revised